subsys/net/lib/nrf_cloud/src/nrf_cloud_rest.c

```diff
@@ -561,16 +580,16 @@ int nrf_cloud_rest_location_get(struct nrf_cloud_rest_context *const rest_ctx,
 	req.body = payload_obj.encoded_data.ptr;
```

Comment:
I am surprised that `do_reply`, `fallback` and `hi_conf` are not sent in the body. Is this imposed by cloud? If not, perhaps `nrf_cloud_obj_location_request_payload_add` can be updated to accept these parameters, instead of encoding them as query strings in the URL?

Reply:
See the link above.